packages/osd-opensearch/src/artifact.js

```diff
@@ -185,10 +185,10 @@ async function getArtifactSpecForSnapshotFromUrl(urlVersion, log) {
 
   // [RENAMEME] Need replacement for other platforms.
   // issue: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/475
-  const platform = process.platform === 'win32' ? 'windows' : process.platform;
+  const platform = process.platform === 'win32' ? 'windows' : 'linux';
```

Comment:
I'm unsure if we should do this. When I added the conditional to check if the platform was Linux because we currently only have Linux-based snapshots available, which is a bad thing. Ideally we should have snapshots for majority of platforms. 

Also, to me this will always be set to Linux no matter which platform and then download the snapshot for Linux. So in the future, if we do have snapshots for Windows, RPM, DEB, then it will still pull the same Linux snapshot and cause confusion. 

I think we should keep this logic this same and perhaps update the CLI error to say use `TEST_OPENSEARCH_FROM`. But I am open for discussion.